readthedocs/restapi/templates/restapi/log.txt

```diff
@@ -0,0 +1,13 @@
+RTD build information start
+Build: {{ build.pk }}
+Project: {{ build.project.slug }}
+Version: {{ build.version.verbose_name }} ({{ build.commit }})
+Date: {{ build.date }}
+Success: {{ build.success }}
+RTD build information end
+
+{% for command in build.commands.all %}
+[rtd-command-info] start-time: {{ command.start_time }}, end-time: {{ command.end_time }}, duration: {{ command.run_time }}, exit-code: {{ command.exit_code }}
```

Comment:
When I read this issue I was thinking just on plain simple txt file, but then you added some metadata to it (times, build id, etc). So, now I'm confused about what we want:

* raw and simple txt plain text
* ini style file
* a mix of both --what you already implemented
* json file (we already have this I think)

What do you think?

Reply:
First I was doing just some plain commands, but them I realized that would be more helpful to have this metadata when viewing the logs, so I took some inspiration from travis https://api.travis-ci.org/v3/job/339760225/log.txt (probably I should mention this early, sorry). There is a json, but contains a lot of information and is hard to read.

Reply:
Oh! I see. It's a good idea to follow other's people ideas here.

I noticed that they also uses colored output, although not necessary, could be a good enhancement maybe later in another PR.